test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,64 @@
+./test_scripts/API/SetAudioStreamingIndicator/023_ATF_N_TC_SetAudioStreamingIndicator_param_data_nonexisting_value_comm_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_with_nonexisting_value_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/024_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_comm_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_empty_value_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/025_ATF_N_TC_SetAudioStreamingIndicator_param_data_missing_parameter_media_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_missing_media_app
+./test_scripts/API/SetAudioStreamingIndicator/026_ATF_N_TC_SetAudioStreamingIndicator_param_data_missing_parameter_non-media_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_missing_non-media_app
+./test_scripts/API/SetAudioStreamingIndicator/027_ATF_N_TC_SetAudioStreamingIndicator_param_data_missing_parameter_navi_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_missing_navi_app
+./test_scripts/API/SetAudioStreamingIndicator/028_ATF_N_TC_SetAudioStreamingIndicator_param_data_missing_parameter_comm_app_SDL_replies_INVALID_DATA.lua Check_that_SDL_responds_with_resultcode_INVALID_DATA_incase_SetAudioStreamingIndicator_is_missing_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/029_ATF_P_TC_SetAudioStreamingIndicator_valid_PLAY_PAUSE_media_app_HMI_replies_SUCCESS.lua Check_that_SDL_responds_with_resultcode_SUCCESS_from_HMI_incase_SetAudioStreamingIndicator_is_correct_valid_PLAY_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/030_ATF_P_TC_SetAudioStreamingIndicator_valid_PLAY_media_app_HMI_replies_SUCCESS.lua Check_that_SDL_responds_with_resultcode_SUCCESS_from_HMI_incase_SetAudioStreamingIndicator_is_correct_valid_PLAY
+./test_scripts/API/SetAudioStreamingIndicator/031_ATF_P_TC_SetAudioStreamingIndicator_valid_PAUSE_media_app_HMI_replies_SUCCESS.lua Check_that_SDL_responds_with_resultcode_SUCCESS_from_HMI_incase_SetAudioStreamingIndicator_is_correct_valid_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/032_ATF_P_TC_SetAudioStreamingIndicator_valid_PLAY_PAUSE_media_app_HMI_replies_SUCCESS_fake_param.lua Check_that_SDL_responds_with_resultcode_SUCCESS_from_HMI_incase_SetAudioStreamingIndicator_is_correct_valid_PLAY_PAUSE_param_other_RPC
+./test_scripts/API/SetAudioStreamingIndicator/033_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_resultcode_success_false Check_that_SDL_responds_with_resultcode_from_HMI_incase_SetAudioStreamingIndicator_is_correct_loop_all_result_codes_success_false
```

Comment:
@dboltovskyi, done in 15f90bf671b71b15b789386c53a516aa989a5169.
Also updated number of last two scripts to 065 / 066